base_link: 'base_link'
hand:
  #The end effector name for grasping
  end_effector_name: 'hand'
  
  # Gripper joints
  joints: ['Gripper_Idol_Gear_Joint', 'Gripper_Servo_Gear_Joint']
  
  #Posture of grippers before grasping
  pregrasp_posture: [0.048, 0.048]
  pregrasp_time_from_start: 4.0
  grasp_posture: [0.016, 0.016]
  grasp_time_from_start: 4.0
  postplace_time_from_start: 4.0
  
  # Desired pose from end effector to grasp [x, y, z] + [R, P, Y]
  grasp_pose_to_eef: [-0.12, 0.0, 0.0]
  grasp_pose_to_eef_rotation: [0.0, 0.0, 0.0]
  end_effector_parent_link: 'wrist_roll_link'
